### PR TITLE
[1.5] Fix local install, add integration tests

### PR DESCRIPTION
**Pull request by ostromart**
Manual port of https://github.com/istio/istio/pull/22411.

### CODE REVIEW

File: manifests/istio-control/istio-autoinject/templates/mutatingwebhook.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- $ca := genCA "istio-sidecar-injector-ca-{{ .Release.Namespace }}" 3650 }}
-{{- if not .Values.global.operatorManageWebhooks }}
+{{- if and (not .Values.global.operatorManageWebhooks) (not .Values.global.istiod.enabled) }}
```
Comment (howardjohn) [inline manifests/istio-control/istio-autoinject/templates/mutatingwebhook.yaml]: Why is this changing? Bad merge?

File: manifests/istio-operator/templates/clusterrole.yaml [hunk 2]
```diff
@@ -0,0 +1,113 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (howardjohn) [inline manifests/istio-operator/templates/clusterrole.yaml]: why is this added?

File: manifests/istio-control/istio-autoinject/templates/mutatingwebhook.yaml (hunk 1 above)
Comment (ostromart) [reply manifests/istio-control/istio-autoinject/templates/mutatingwebhook.yaml]: I had to fix this manually - it's not related to this PR but a test is failing here that I think was passing elsewhere because of a race. Without this change, we can have two webhook configs that race.

File: manifests/istio-operator/templates/clusterrole.yaml (hunk 2 above)
Comment (ostromart) [reply manifests/istio-operator/templates/clusterrole.yaml]: In the original PR it is a move, but for 1.5 I'm keeping the original templates in case someone relies on the old path, which is why this shows up as an add, not move.

**Comment by ostromart** [comment]
ping @howardjohn

File: manifests/istio-operator/templates/clusterrole.yaml (hunk 2 above)
Comment (howardjohn) [reply manifests/istio-operator/templates/clusterrole.yaml:1]: Why is this needed even in the original pr? how does the operator control impact local install (istioctl)? is it for `istioctl operator init`?
